README.md

```diff
@@ -34,6 +34,15 @@ This library brings the rich assertion types of Google's C++ testing library
 > not be used directly. Those items or modules are only for internal uses and
 > their API may change without a major version update.
 
+## Learning resources
+
+If you're just getting started with `googletest`, consider going through
+the first chapter of
+["Advanced testing for Rust applications"](https://github.com/mainmatter/rust-advanced-testing-workshop),
```

Comment:
It would perhaps be better to link directly to the relevant directory — when I look at the root of the repository, I see instructions for installing stuff on my machine, but I don't see instructions on how to get to read the exercises without those tools.

```suggestion
["Advanced testing for Rust applications"](https://github.com/mainmatter/rust-advanced-testing-workshop/tree/main/exercises/01_better_assertions),
```

@LukeMathWalker, it might be nice for you to add a README file to that directory (and the other ones) so that one can browser GitHub and comfortably read the material that way.

Reply:
> when I look at the root of the repository, I see instructions for installing stuff on my machine, but I don't see instructions on how to get to read the exercises without those tools.

The tools are strictly required to perform the exercises, that's why I chose to link to the repository instead of the exercise directory.
Especially when "testing testing", we need the learner to write tests _that fail_ and verify that they fail as expected. That's what the tooling (`ctr` and `wr`) take care of, which wouldn't work with a vanilla `cargo test` since the expectation is not to get the tests to pass.

Reply:
> The tools are strictly required to perform the exercises, that's why I chose to link to the repository instead of the exercise directory.

I see! It just makes the material less useful for people who want to read it in isolation — but that's of course a reasonable choice to make!

Thanks for explaining.